### DISCUSSION TITLE
Support for Svelte4 in @imask/svelte ?

### DISCUSSION THREAD

**Discussion by gterras** [uNmAnNeR/imaskjs#889] [General]
Hi, 

Thanks for the awesome lib! I know it's a recent release but do you plan to support Svelte4 ? It looks like forcing the update breaks imask.

**Comment by uNmAnNeR** [comment]
I supposed that it should work out of the box as long as they say it's mostly about maintenance. But i will check, thanks for the feedback

**Comment by uNmAnNeR** [comment]
should work in v7.1.2

**Reply by gterras** [reply]
Awesome thank you!
